driving_gridworld/gridworld.py

```diff
@@ -12,10 +12,12 @@ def __init__(self,
                  headlight_range=5,
                  num_bumps=3,
                  num_pedestrians=3,
+                 num_car_obstacles=1,
```

Comment:
This is good, but can you add the same argument to `exe/road.py`, please, so that it can be set from the command line?